docs/application/index.md

```diff
@@ -1,178 +1,323 @@
+						<a class="docs-read-more" href="vstools/index.md" style="float:left;" target="_self">Learn more</a>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="column docs-column content-box-column content-box-column-1 content-box-hover">
+			<div class="col content-box-wrapper content-wrapper-background link-type-text content-icon-wrapper-yes icon-wrapper-hover-animation-pulsate" data-animationoffset="100%" style="background-color:#ffffff;">
+				<div class="heading-with-icon">
+					<div class="image"><a href="https://code.visualstudio.com/?wt.mc_id=DX_841432" rel="noopener noreferrer" target="_blank"><img alt="" height="32" src="https://visualstudio.microsoft.com/wp-content/uploads/2019/09/vs-code-responsive-01-1.png" width="32" /></a></div>
+					<p><a href="vscode-ext/index.md" target="_self">Visual Studio Code Extension for Tizen</a></p>
+				</div>
+				<div class="docs-content-container" style="color:#333333; height:300px;">
+					<div class="vscom-boxes-body">
+						<p><img alt="" class="vscom-boxes-image x-hidden-focus" src="./media/vscode-new-hero.png" /></p>
+						<p class="vscom-boxes-text">Develop Tizen .NET applications using Visual Studio Code on any OS.</p>
+						<a class="docs-read-more" href="vscode-ext/index.md" style="float:left;" target="_self">Learn more</a>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="column docs-column content-box-column content-box-hover content-box-column-last content-box-column-last-in-row">
+			<div class="col content-box-wrapper content-wrapper-background link-type-text content-icon-wrapper-yes icon-wrapper-hover-animation-pulsate" data-animationoffset="100%" style="background-color:#ffffff;">
+				<div class="heading-with-icon">
+					<div class="image"><a href="https://visualstudio.microsoft.com/vs/mac/" target="_self"><img alt="" height="32" src="https://visualstudio.microsoft.com/wp-content/uploads/2019/09/VSforMacIcon2019-2.svg" width="32" /></a></div>
+					<p><a href="vstools-mac/overview.md" target="_self">Visual Studio for Mac Extension for Tizen</a></p>
+				</div>
+				<div class="docs-content-container" style="color:#333333; height:300px;">
+					<div class="vscom-boxes-body">
+						<p><img alt="" class="vscom-boxes-image" src="./media/vsmac-new-hero.png" /></p>
+						<p class="vscom-boxes-text">Develop Tizen .NET applications easily with Visual Studio for Mac.</p>
+						<a class="docs-read-more" href="vstools-mac/overview.md" style="float:left;" target="_self">Learn more</a>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+<div class="docs-ui-get-started">
+	<div class="docs-content-boxes">
+		<p style="text-align: right; margin-right: 20px">Image source: <a href="https://visualstudio.microsoft.com/" target="_blank">https://visualstudio.microsoft.com/</a></p>
+	</div>
+</div>
 
-To see the trademark notice list, see [Trademarks](../trademarks.md).
+## Tizen Studio
```

Comment:
Question: Is there a reason to keep Tizen Studio in the end?

Could we please swap it with Tizen Extension for Visual Studio Family? If we swap, then the "Image source" could also be left aligned and would look well.

Reply:
> Question: Is there a reason to keep Tizen Studio in the end?

=> Recently, C# has been more popular.

> Could we please swap it with Tizen Extension for Visual Studio Family? If we swap, then the "Image source" could also be left aligned and would look well.

=> Image source has to be right-aligned.
If you see index.md alone, it can be weird.
However, if you see https://docs1.stg.tizen.org/staging/1150/application/, you would think the current position is perfect. :blush: